src/World.cpp

```diff
@@ -624,48 +721,39 @@ void cWorld::GenerateRandomSpawn(void)
 
 bool cWorld::CheckPlayerSpawnPoint(int a_PosX, int a_PosY, int a_PosZ)
 {
-	// The bottom layer cannot hold a valid spawn point
-	if (a_PosY <= 0)
+	// Check height bounds
+	if ((a_PosY <= 0) || (a_PosY > 255))
```

Comment:
I didn't have this when I wrote it, though at the time I remember thinking 'I wish there was a method for this, magic numbers suck'.

I shall change it later today.